### ISSUE TITLE
rowId doesn't work

### ISSUE THREAD

**Issue by orekav** [iamisti/mdDataTable#199]
In the demo -> Ajax html cell support

I put:

        <mdt-custom-cell column-key="protein">
            <span ng-class="{'red': value > 5, 'green': value <= 5}">{{value}}</span>
                      <span>This is the row id for this column: {{rowId}}</span>

        </mdt-custom-cell>

And nothing appear in the place of {{rowId}} , also... appears '0' if i write {{row-id}}

**Comment by iamisti** [comment]
Hi @orekav , did you also set `table-row-id-key`?

Example:
```html
<mdt-table
    mdt-row="{
        'data': yourDataArray,
        'table-row-id-key': 'id',
        'column-keys': ['name', 'calories', 'fat', 'carbs', 'protein', 'sodium', 'calcium', 'iron']
    }">

    <mdt-header-row>
        <mdt-column>Dessert (100g serving)</mdt-column>
        <mdt-column>Type</mdt-column>
        <mdt-column>Calories</mdt-column>
        <mdt-column sortable-rows-default>Fat (g)</mdt-column>
        <mdt-column>Carbs (g)</mdt-column>
        <mdt-column>Protein (g)</mdt-column>
    </mdt-header-row>

    <mdt-custom-cell column-key="protein">
        <span ng-class="{'red': value > 5, 'green': value <= 5}">{{value}}</span>
                  <span>This is the row id for this column: {{rowId}}</span>
    </mdt-custom-cell>
</mdt-table>
```

**Comment by orekav** [comment]
I said that I had use you your own DEMO -> http://codepen.io/iamisti/pen/pbLYwq

**Comment by iamisti** [comment]
Found the issue. It was working with `mdt-row-paginator` but it wans't when you use `data` property with `mdt-row` attribute.

Fixed with `v1.8.8`

Thanks for the ticket, was very useful!

**Comment by gerome0123** [comment]
@iamisti your example in rowId is not working???
